packages/ord-connect/src/components/SelectWalletModal/style.css

```diff
@@ -152,24 +157,43 @@
 }
 
 .ord-connect-wallet-modal .wallet-option-button:first-child {
-  padding: 32px 32px 24px;
+  padding: 24px 24px 16px;
   border: 0px;
   margin-bottom: 0px;
-  border-radius: 20px 20px 20px 20px;
+  border-radius: 20px 20px 0px 0px;
 }
 
 .ord-connect-wallet-modal .wallet-option-button:nth-of-type(2) {
   border: 0px;
-  padding: 24px 32px 32px;
+  padding: 16px 24px 24px;
   margin-bottom: 0px;
   border-radius: 0px 0px 20px 20px;
 }
 
-.ord-connect-wallet-modal .wallet-option-label {
-  margin-left: 16px;
+.ord-connect-wallet-modal .wallet-option {
   flex-grow: 1;
+  margin-left: 16px;
   text-align: left;
 }
+.ord-connect-wallet-modal .wallet-option-label {
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 20px;
+}
+
+.ord-connect-wallet-modal .wallet-option-info {
+  display: block;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+  color: #8c8c8c;
+}
+
+@media (min-width: 768px) {
+  .ord-connect-wallet-modal .wallet-option-info {
+    display: none;
```

Comment:
i think min-width is not enough to determine for mobile now, because ipad air is `820 x 1180`

Reply:
<img width="622" alt="image" src="https://github.com/ordzaar/ord-connect/assets/40191153/3ad2d0d8-2f43-4fd7-888a-81a7727e36b4">